### PR TITLE
docs: update docs for tf.python.tools.import_pb_to_tensorboard

### PR DESCRIPTION
**Pull request by howl-anderson**
Using SavedModel instead .pb model

### CODE REVIEW

File: tensorflow/python/tools/import_pb_to_tensorboard.py [hunk 1]
```diff
@@ -41,19 +41,19 @@
 
 
 def import_to_tensorboard(model_dir, log_dir, tag_set):
-  """View an imported protobuf model (`.pb` file) as a graph in Tensorboard.
+  """View an SavedModel as a graph in Tensorboard.
```
Comment (ashutosh1919) [inline tensorflow/python/tools/import_pb_to_tensorboard.py]: imported/saved protobuf model (`.pb` file)

File: tensorflow/python/tools/import_pb_to_tensorboard.py [hunk 2]
```diff
@@ -41,19 +41,19 @@
...
 
   Args:
-    model_dir: The location of the protobuf (`pb`) model to visualize
+    model_dir: The directory containing the SavedModel to import.
```
Comment (ashutosh1919) [inline tensorflow/python/tools/import_pb_to_tensorboard.py]: directory containing the imported/saved protobuf (`pb`) model to visualize.

File: tensorflow/python/tools/import_pb_to_tensorboard.py [hunk 3]
```diff
@@ -41,19 +41,19 @@
...
     log_dir: The location for the Tensorboard log to begin visualization from.
     tag_set: Group of tag(s) of the MetaGraphDef to load, in string format,
         separated by ','. For tag-set contains multiple tags, all tags must be
         passed in.
 
   Usage:
-    Call this function with your model location and desired log directory.
+    Call this function with your SavedModel location and desired log directory.
```
Comment (ashutosh1919) [inline tensorflow/python/tools/import_pb_to_tensorboard.py]: imported/saved model

File: tensorflow/python/tools/import_pb_to_tensorboard.py [hunk 4]
```diff
@@ -41,19 +41,19 @@
...
     Launch Tensorboard by pointing it to the log directory.
-    View your imported `.pb` model as a graph.
+    View your imported SavedModel as a graph.
```
Comment (ashutosh1919) [inline tensorflow/python/tools/import_pb_to_tensorboard.py]: imported/saved `.pb` model

**Review by ashutosh1919 — CHANGES_REQUESTED** [review]

File: tensorflow/python/tools/import_pb_to_tensorboard.py (hunk 1 above)
Comment (howl-anderson) [reply tensorflow/python/tools/import_pb_to_tensorboard.py]: @ashutosh1919 Thank you for your suggestion, however, this function will try to load model_dir as SavedModel (expected `saved_model.pb` or `saved_mode.pbtxt` inside it), not a protobuf model (.pb file).

File: tensorflow/python/tools/import_pb_to_tensorboard.py (hunk 2 above)
Comment (howl-anderson) [reply tensorflow/python/tools/import_pb_to_tensorboard.py]: ditto

File: tensorflow/python/tools/import_pb_to_tensorboard.py (hunk 4 above)
Comment (howl-anderson) [reply tensorflow/python/tools/import_pb_to_tensorboard.py]: ditto

File: tensorflow/python/tools/import_pb_to_tensorboard.py (hunk 3 above)
Comment (howl-anderson) [reply tensorflow/python/tools/import_pb_to_tensorboard.py]: ditto

**Comment by howl-anderson** [comment]
@ringw Please have a look at this PR, thanks!

**Review by mihaimaruseac — COMMENTED** [review]
It seems 3dc97bb16d797ef9002a03f4ab0ffdbdf88e851b#diff-9291bb0bf6f25c9f46e445f544a2cc7f changed the behavior from importing from protobuf to importing from a `SavedModel`.

In this case, it would be better to update the entire file (there is a docstring at the beginning of the file, from example) and rename it accordingly.

**Comment by mihaimaruseac** [comment]
Adding @nfelt (from Tensorboard side), @k-w-w (from `SavedModel` side) and @akshaym (afaik, owner of this code) for more review

**Comment by nfelt** [comment]
cc @davidsoergel for TensorBoard since this is relevant to https://github.com/tensorflow/tensorboard/issues/1962.  I don't have any particular context on this script or why it lives here versus in TensorBoard proper, other than that it's been around for a long time.

**Comment by mihaimaruseac** [comment]
We can also move the file in this PR (plus one on Tensorboard side), if that is case

**Comment by mihaimaruseac** [comment]
Closed by mistake, reopening

**Review by k-w-w — APPROVED** [review]
Thanks for clarify these args! They do expect the model dir, not the path to the pb file.

**Comment by ringw** [comment]
The other reviewers have much more context here, so I'm removing myself.

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**